spec/requests/api/virtual_templates_spec.rb

```diff
@@ -0,0 +1,183 @@
+#
+# REST API Request Tests - Virtual Template Specs
+#
+# - Creating a virtual_template               /api/virtual_templates                POST
+# - Creating a virtual_template via action    /api/virtual_templates                action "create"
+# - Edit a virtual_template                   /api/virtual_templates/:id            action "edit"
+# - Edit multiple virtual_templates           /api/virtual_templates                action "edit"
+# - Delete a virtual_template by action       /api/virtual_templates/:id            action "delete"
+# - Delete multiple virtual_templates         /api/virtual_templates                action "delete"
+# - Delete virtual_template                   /api/virtual_templates                DELETE
+# - Get list of virtual templates             /api/virtual_templates                GET
+# - Get a virtual template                    /api/virtual_templates/:id            GET
+
+RSpec.describe 'Virtual Template API' do
+  context 'virtual templates index' do
+    it 'can list all the virtual templates' do
+      api_basic_authorize collection_action_identifier(:virtual_templates, :read, :get)
+      FactoryGirl.create(:virtual_template)
```

Comment:
Here is a reason why I don't like amazon being the default.  It's very not clear in the test here that virtual_template implies that it's amazon.